### PR TITLE
fix: Caddyfile and default JS deps for the PWA

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| Tickets       | fixes https://github.com/api-platform/api-platform/issues/1726
| License       | MIT
| Doc PR        | todo

### CODE REVIEW

**Comment by LeoBenoist** [comment]
@dunglas this is working for me for next. 
⚠️ I don't know if you want to support by default PWA.

For PWA it would require to add an exception for the  manifest.json `|| {path}.startsWith("/manifest.json")` It is the most commonly used URL in documentation around the web (including Mozzila and Google).

**Comment by dunglas** [comment]
Good catch, I'll add it!

**Comment by LeoBenoist** [comment]
You’re welcome :). Thank you very much for your work and API platform. On my side it will power a small startup that has for goal to make the world a better place in Africa.

File: pwa/pages/index.js [hunk 1]
```diff
@@ -58,7 +59,7 @@ const Welcome = () => (
               </div>
               <div className="other__content">
                 <h3>
-                  <a href="/docs">API</a>
+                  <Link href="/docs">API</Link>
```
Comment (LeoBenoist) [inline pwa/pages/index.js:62]: A small one here. With next/link I think it should be `<Link href="/docs"><a>API<a/a></Link>`. Link is only a wrapper on next contrary to react router. (false friend)

**Pull request merged**